package.json

```diff
@@ -2,50 +2,49 @@
   "name": "ember-sortable",
   "version": "1.9.4",
   "description": "Sortable UI primitives for Ember.",
+  "keywords": [
+    "ember-addon",
+    "sortable"
+  ],
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "https://github.com/jgwhite/ember-sortable",
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": "https://github.com/jgwhite/ember-sortable",
-  "engines": {
-    "node": ">= 0.12.0"
+  "dependencies": {
+    "ember-cli-babel": "^6.6.0",
+    "ember-cli-htmlbars": "^1.0.10",
+    "ember-invoke-action": "^1.4.0",
+    "ember-new-computed": "^1.0.2"
```

Comment:
I think this is only for Ember 1.11 and earlier, so I wonder if this could be dropped at this point?